src/qt/mempoolstats.cpp

```diff
@@ -135,16 +140,47 @@ void MempoolStats::drawChart()
         allDataLabel->setFont(QFont(LABEL_FONT, LABEL_KV_SIZE, QFont::Light));
     }
 
-    static const QString checkbox_style_base = QStringLiteral("background-color:rgb(255,255,255);");
-    cbShowNumTxns->setStyleSheet(cbShowNumTxns->isChecked() ? (checkbox_style_base + "color:rgb(188,49,62);") : checkbox_style_base);
-    cbShowMinFeerate->setStyleSheet(cbShowMinFeerate->isChecked() ? (checkbox_style_base + "color:rgb(49,113,62);") : checkbox_style_base);
-    cbShowMemUsage->setStyleSheet(cbShowMemUsage->isChecked() ? (checkbox_style_base + "color:rgb(15,68,113);") : checkbox_style_base);
-
-    last10MinLabel->setEnabled((timeFilter == TEN_MINS));
-    lastHourLabel->setEnabled((timeFilter == ONE_HOUR));
-    lastDayLabel->setEnabled((timeFilter == ONE_DAY));
-    allDataLabel->setEnabled((timeFilter == 0));
-
+    // Detect dark mode for color palette selection
+    const QColor bg_colour = palette().color(backgroundRole());
+    const bool dark_mode = GUIUtil::isDarkMode(bg_colour);
+    
+    // Define color palettes for dark and light themes
+    const QColor color_orange = dark_mode ? QColor(247, 147, 26) : QColor(216, 92, 1);
+    const QColor color_green = dark_mode ? QColor(69, 222, 181) : QColor(0, 125, 50);
+    const QColor color_blue = dark_mode ? QColor(137, 170, 255) : QColor(2, 61, 204);
+    
+    // Line colors with alpha for the graph
+    const QColor line_orange = dark_mode ? QColor(247, 147, 26, 250) : QColor(216, 92, 1, 250);
+    const QColor line_green = dark_mode ? QColor(69, 222, 181, 250) : QColor(0, 125, 50, 250);
+    const QColor line_blue = dark_mode ? QColor(137, 170, 255, 250) : QColor(2, 61, 204, 250);
```

Comment:
I think we can deduplicate these since color_\* only get used for `.name()` and that ignores alpha (we can make it explicit with `.name(QColor::HexRgb)`)